tests/helpers/database.ts

```diff
@@ -35,6 +36,22 @@ export const dbHelper = () => {
     await mongoClient.close()
   }
 
+  const dbCreateWithClient = async (...fns: any): Promise<any> => {
+    console.log(`Client created`)
+    const mongoClient = await getMongoClient()
+
+    let result: any
+
+    try {
+      const db = mongoClient.db()
+      result = await pipeP(...fns)(db)
+    } catch (err) {
+      console.log(err)
+    }
+    await delay(1300)
+    return { result, mongoClient }
```

Comment:
FYI: mongoClient cannot be closed until after the test runs, so I'm passing this back.